src/editor/editorui.js

```diff
@@ -60,10 +54,93 @@ export default class EditorUI {
+		 * @private
+		 * @member {module:ui/editorui/editoruiview~EditorUIView} #_view
+		 */
+		this._view = view;
+
+		// Check if `view` parameter was passed. It is deprecated and should not be used.
+		if ( view ) {
+			/**
+			 * This error is thrown when  the deprecated `view` parameter is passed to the
+			 * {@link module:core/editor/editorui~EditorUI#constructor EditorUI constructor}. Only subclass (for example
+			 * {@link module:editor-classic/classiceditorui~ClassicEditorUI}) should use it without passing it further.
+			 *
+			 * @error deprecated-editorui-view-param-in-constructor
+			 */
+			log.warn( 'deprecated-editorui-view-param-in-constructor: The EditorUI#constructor `view` parameter is deprecated.' );
+		}
+
 		// Informs UI components that should be refreshed after layout change.
 		this.listenTo( editor.editing.view.document, 'layoutChanged', () => this.update() );
 	}
 
+	/**
+	 * **Deprecated** since `v12.0.0`. This property is deprecated and should not be used. Use the property
+	 * from the subclass directly instead, for example
+	 * {@link module:editor-classic/classiceditorui~ClassicEditorUI#view ClassicEditorUI#view}.
+	 *
+	 * The main (top–most) view of the editor UI.
+	 *
+	 * @deprecated v12.0.0 This property is deprecated and should not be used. Use the property
+	 * from the subclass directly instead, for example
+	 * {@link module:editor-classic/classiceditorui~ClassicEditorUI#view ClassicEditorUI#view}.
+	 * @readonly
+	 * @member {module:ui/editorui/editoruiview~EditorUIView} #view
+	 */
+	get view() {
+		/**
+		 * This error is thrown when a component tries to access deprecated
+		 * {@link module:core/editor/editorui~EditorUI#element `EditorUI view`} property. Instead the `view` property
+		 * from the subclass (for example {@link module:editor-classic/classiceditorui~ClassicEditorUI#view ClassicEditorUI#view})
+		 * should be accessed directly.
+		 *
+		 * @error deprecated-editorui-view
+		 */
+		log.warn( 'deprecated-editorui-view: The EditorUI#view property is deprecated.' );
+
+		return this._view;
+	}
+
+	/**
+	 * The main (outermost) DOM element of the editor UI.
+	 *
+	 * For example, in {@link module:editor-classic/classiceditor~ClassicEditor} it is a `<div>` which
+	 * wraps the editable element and the toolbar. In {@link module:editor-inline/inlineeditor~InlineEditor}
+	 * it is the editable element itself (as there is no other wrapper). However, in
+	 * {@link module:editor-decoupled/decouplededitor~DecoupledEditor} it is set to `null` because this editor does not
+	 * come with a single "main" HTML element (its editable element and toolbar are separate).
+	 *
+	 * This property can be understood as a shorthand for retrieving the element that a specific editor integration
+	 * considers to be its main DOM element.
+	 *
+	 * @readonly
+	 * @member {HTMLElement|null} #element
+	 */
+	get element() {
+		return null;
```

Comment:
I think you do not need to define the body of this property here. It should be enough to define just the docs for it.